duperemove.c

```diff
@@ -183,6 +186,7 @@ static int parse_options(int argc, char **argv)
 		{ "lookup-extents", 1, NULL, LOOKUP_EXTENTS_OPTION },
 		{ "one-file-system", 0, NULL, ONE_FILESYSTEM_OPTION },
 		{ "hash", 1, NULL, HASH_OPTION },
+		{ "skip-zeroes", 1, NULL, SKIP_ZEROES_OPTION },
```

Comment:
Want to do a '0' for the 2nd field here as skip-zeroes does not take an argument. This is probably why it only worked one way and not the other :)